dev-docs/modules/iabCatalogTranslation.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page_v2
+page_type: module
+title: Module - IAB Category Translation
+description: Converts between ad agency brand categories and IAB brand categories.
+module_code : CategoryTranslation
+display_name : CategoryTranslation
+enable_download : true
+sidebarType : 1
+---
+
+# IAB Category Translation
+
+{:.no_toc}
+
+This module supports the mapping between ad agency brand categories and IAB subcategories and conversion of IAB subcategories to Freewheel industry group identifiers. The Freewheel identifiers ensure competitve separation of industries and products. 
+
+Each bid request must return one [IAB subcategory](https://support.aerserv.com/hc/en-us/articles/207148516-List-of-IAB-Categories).
+
+The module provides the following: 
+
+- Provides an API that converts the demand providers own ad categories into IAB subcategories. If a demand provider elects to use Prebid API for this process, they must include a path to a local mapping file that converts the ad server categories to IAB subcatgories. 
+
+- Convert IAB subcategories to a Freewheel industry group identifier.
+
+## How to use the module:
+
+1. A Prebid.js package is built that contains this module and the [Freewheel]({site.github.url}}/dev-docs/module/freewheel.html) module.  
+2. The inclusion of this module causes Prebid to download a mapping file to local storage. The user also has the option to provide their own mapping file. 
+3. At runtime, brand category translation happens as needed. 
+
+
+## Using A Custom Map File
+The IAB Category Translation module uses a default mapping file to convert adserver categories to IAB sub categories . If a publisher prefers to use their own mapping file they will need to set the URL location of that file. They can do so by adding the following to their Prebid.js configuration: 
+
+```
+pbjs.setConfig({
+    "brandCategoryTranslation": {
+       "translationFile": "<url_to_file>"
+    }
+});
+```
+This file will be stored locally to expedite the conversion process. If a publisher opts to not provide a conversion mapping file Prebid will use its default conversion mapping file. 
+
+## Adapter Integration
+
+Adapters need to return IAB subcategories in their repsonses. For adapters who do not have that capability they will provide a mapping file in JSON format that will convert their proprietary brand categories to IAB subcategories. 
+```
```

Comment:
Add an extra link between the text and the code-block; on the locally ran site, half of this code is appearing as plain-text instead of in the code-block.

Reply:
This threw me for a bit until I realized link was a typo for line. Strange that the code block broke for you. In my local version and in markdown editor it looked fine. I've revised it, let me know if you are still having issues. Out of curiosity what browser were you using?